DeUrgenta/Utils/Extensions/FontManager.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+import SwiftUI
+
+enum IBMPlexSansFonts: String {
+    case IBMPlexSansRegular = "IBMPlexSans-Regular"
+    case IBMPlexSansMedium = "IBMPlexSans-Medium"
+    case IBMPlexSansSemiBold = "IBMPlexSans-SemiBold"
+    case IBMPlexSansBold = "IBMPlexSans-Bold"
+}
+
+struct CustomFont: ViewModifier {
+    @Environment(\.sizeCategory) var sizeCategory
```

Comment:
Not clear to me what is the purpose of `sizeCategory`?

Reply:
Yes, we don't need sizeCategory